tern/report/report.py

```diff
@@ -61,7 +61,7 @@ def generate_format(images, format_string):
             name=format_string,
             invoke_on_load=True,
         )
-        return mgr.driver.generate(images)
+        return mgr.driver.generate(images, print_layer, print_inclusive)
```

Comment:
Small nit: I think we should be consistent with the variable names. We use `layer_inclusive` throughout most of these changes and `print_inclusive` here.

Reply:
Those args should've been removed in last commit as the args.layer and args.layer_inclusive are later Image.until_layer Image.only_layer